packages/py-moose-lib/tests/test_olap_dictionary.py

```
@@ -0,0 +1,1647 @@
+        open(os.path.join(app_pkg, "__init__.py"), "w").close()
+        with open(os.path.join(app_pkg, "main.py"), "w") as f:
+            f.write(main_py)
+        infra_map = _run_serializer(tmp, _moose_lib_root())
+
+    d = infra_map["olapDictionaries"]["dict_items_optional"]
+    assert d["comment"] == "Test dictionary"
+    assert d["database"] == "mydb"
+    assert d["clusterName"] == "my_cluster"
+    assert d["settings"]["max_threads"] == "2"
+    iq = d.get("invalidateQuery")
+    assert iq is not None and "max" in iq and "updated_at" in iq
```

Ruff (0.15.11)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)